src/main/java/frc/robot/RobotContainer.java

```diff
@@ -420,6 +468,22 @@ private void limelightRotationOff() {
     // m_rotationSupplier));
   }
 
+  private void hangarAlignOn() {
+    _hangarAlign = true;
+  }
+
+  private void hangarAlignOff() {
+    _hangarAlign = false;
+  }
+  
+  private void intakeDown() {
+    intakeDown = true;
+  }
+
+  private void intakeUp() {
+    intakeDown = false;
+  }
```

Comment:
Why are you adding this intake control? this should be managed in commands and default commands.

Reply:
it doesn't control intake, the boolean just monitors the state of the intake, so that if intake is not down the operator joystick doesn't do anything